src/parser/core/modules/CastTime.ts

```diff
@@ -7,7 +7,8 @@ import {Data} from './Data'
 
 export interface CastTimeAdjustment {
 	actions: number[] | 'all',
-	castTime: number,
+	type: 'time' | 'percentage'
+	adjustment: number,
 	start: number,
 	end: number | null
```

Comment:
realise it's pre-existing, but this should probably be `end?: number`. will tie in well with other suggestions about function sigs